api/v1beta1/infraenv_types.go

```diff
@@ -88,6 +88,10 @@ type InfraEnvSpec struct {
 	// +kubebuilder:default=x86_64
 	// +optional
 	CpuArchitecture string `json:"cpuArchitecture,omitempty"`
+
+	// IPXEScriptType the script type that should be served (HostRedirect/Boot)
+	// +optional
```

Comment:
It seems like effectively the default behavior is to act as if "Boot" is the selection. WDYT about making that the explicit default here?

Reply:
And perhaps also limit the values to these two using a validation marker. https://book.kubebuilder.io/reference/markers/crd-validation.html